src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/InstitutionListingRepository.php

```diff
@@ -105,7 +105,7 @@ public function getInstitutionsForSelectRaa(IdentityId $actorId)
                 InstitutionAuthorization::class,
                 'a',
                 Join::WITH,
-                "i.institution = a.institution AND a.institutionRole IN (:authorizationRoles)"
+                "r.institution = a.institution AND a.institutionRole IN (:authorizationRoles)"
```

Comment:
In the commit message, you state that there is a bug in this query. The bug is that the join was on the wrong table when looking at the code. 

Is my conclusion correct that: We now only return institutions that actually have a RA? As you now join on the RA Listing instead of joining on the Institution listing.

Reply:
Your conclusion is correct if `:roles` = 'ra'